src/pb.sh

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/bin/sh
 
 endpoint="${PB_ENDPOINT:-https://ptpb.pw}"
 jq_args="${PB_JSON:--r .url}"
 private="${PB_PRIVATE:-0}"
 
 
 pb_ () {
-  local command filename
+  command filename
```

Comment:
Removing `local` here changes the meaning of the line; this executes the `command` shell built-in.